README.md

```diff
@@ -1,77 +1,13 @@
 # MongoDB Specifications
 
+[![Documentation Status](https://readthedocs.org/projects/specifications/badge/?version=latest)](http://specifications.readthedocs.io/en/latest/?badge=latest)
+
 This repository holds in progress and completed specification for features of MongoDB, Drivers, and associated products.
 Also contained is a rudimentary system for producing these documents.
 
 ## Driver Mantras
 
-When developing specifications -- and the drivers themselves -- we follow the following principles:
```

Comment:
I moved this to its own file in the source tree since it was referenced from the CMAP spec.